inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/page/workload/WorkloadPageMenuItem.java

```diff
@@ -94,10 +92,12 @@ public boolean applies()
 
         // Visible if the current user is a curator or project admin
         User user = userRepo.getCurrentUser();
+
         return (projectService.isCurator(project, user)
             || projectService.isProjectAdmin(project, user))
             && WebAnnoConst.PROJECT_TYPE_ANNOTATION.equals(project.getMode())
-            && workloadProperties.isWorkloadManagerActive();
+            && !workloadAndWorkflowService.getWorkloadManager(project).equals(
```

Comment:
We normally compare CONSTAT.equals(foo) because it is more likely for foo to be null than CONSTANT